widgets/index-selector/index.js

```diff
@@ -0,0 +1,43 @@
+var React = require('react');
+var cx = require('classnames');
+
+var findIndex = require('lodash/array/findIndex');
+var utils = require('../../lib/widgetUtils.js');
+
+function indexSelector({
```

Comment:
I started doing jsdoc on widgets/, what do you think of adding it.

Two purposes:
- right now helps documenting widgets in code AND readme.md (you can just copy paste the jsdoc)
- in the future will help generating some API docs

Once ready what do you think about doing it for this widget? There's an example in #63